server/src/main/java/org/elasticsearch/search/aggregations/AggregationBuilder.java

```diff
@@ -131,21 +132,6 @@ protected AggregationBuilder doRewrite(QueryRewriteContext queryShardContext) th
         return this;
     }
 
-    /**
```

Comment:
Since we're here, could we add an `@override` annotation to `rewrite()` method above?  Tiny nit, sorry :(

Reply:
:+1: